### PR TITLE
bump dev version on master

### PR DESCRIPTION
**Pull request by munkm**
This should go in after yt-project/yt#2526 is merged

### CODE REVIEW

**Comment by munkm** [comment]
@matthewturk should the number after "dev" be bumped like I've done or is that unnecessary?

**Comment by matthewturk** [comment]
I think it's fine as-is.

**Comment by munkm** [comment]
@Xarthisius I can't see why the docs were failing on this PR? Did I increment the version in a way that broke it?

**Comment by Xarthisius** [comment]
> @Xarthisius I can't see why the docs were failing on this PR? Did I increment the version in a way that broke it?

It was probably a random failure. Ignore it.

File: setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 except pkg_resources.DistributionNotFound:
     pass  # yay!
 
-VERSION = "3.6.dev0"
+VERSION = "3.6.dev1"
```
Comment (Xarthisius) [inline setup.py:37]: Since we released 3.6 shouldn't it be `3.7.dev0` ?

Comment (munkm) [reply setup.py]: wow. Yes. Thank you.

**Pull request merged**